pkg/api/rebalance_req.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import "github.com/pborman/uuid"
+
+// RebalanceStatus represents Rebalance Status
+type RebalanceStatus uint64
```

Comment:
This will need more values. See GD1:

enum gf_defrag_status_t {
        GF_DEFRAG_STATUS_NOT_STARTED,
        GF_DEFRAG_STATUS_STARTED,
        GF_DEFRAG_STATUS_STOPPED,
        GF_DEFRAG_STATUS_COMPLETE,
        GF_DEFRAG_STATUS_FAILED,
        GF_DEFRAG_STATUS_LAYOUT_FIX_STARTED,
        GF_DEFRAG_STATUS_LAYOUT_FIX_STOPPED,
        GF_DEFRAG_STATUS_LAYOUT_FIX_COMPLETE,
        GF_DEFRAG_STATUS_LAYOUT_FIX_FAILED,
};

Reply:
I removed duplicates and wrote this in rest.go file